react/src/components/VFolderTableFormItem.tsx

```diff
@@ -30,6 +30,22 @@ const VFolderTableFormItem: React.FC<VFolderTableFormItemProps> = ({
 }) => {
   const form = Form.useFormInstance();
   const { t } = useTranslation();
+
+  const ownerInfo = form.getFieldValue('owner');
```

Comment:
- `form.getFieldValue` does not guarantee triggering a re-render when the `owner` is changed.

Reply:
Changed the part that gets the owner information to be managed inside `VFolderTableFormItem.tsx`. Modified the dependency settings to determine if the owner information is valid when it is changed and, if valid, to initialize the existing assigned vfolder alias.